util/tracer/tracer_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tobias Schottdorf (tobias.schottdorf@gmail.com)
+
+package tracer
+
+import (
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cockroachdb/cockroach/util"
+)
+
+type traceID int
+
+func (id traceID) TraceID() string {
+	return strconv.Itoa(int(id))
+}
+
+func (id traceID) TraceName() string {
+	return id.TraceID()
+}
+
+func TestTracer(t *testing.T) {
```

Comment:
yes, more tests will be coming when I feel moderately confident that things are somewhat stable (moving to protos and all).